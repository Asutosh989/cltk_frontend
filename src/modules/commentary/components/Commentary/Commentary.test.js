import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import Commentary from './Commentary';

describe('Commentary', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<Commentary
						comment={{
							index: 0,
						}}
					/>
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
