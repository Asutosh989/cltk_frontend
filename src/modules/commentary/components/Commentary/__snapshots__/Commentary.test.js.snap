// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Commentary renders correctly 1`] = `
<div
  className="meta-item panel-item commentary-comment "
  data-num={0}
>
  <div
    className="show-more-toggle"
    onClick={[Function]}
  >
    <i
      className="mdi mdi-plus paper-shadow"
    />
    <i
      className="mdi mdi-minus paper-shadow"
    />
  </div>
  <div
    className="comment-meta"
  >
    <span
      className="comment-ref"
    >
      Â 
    </span>
    <span
      className="comment-authors"
    />
    <span
      className="comment-date"
    />
  </div>
  <p
    className="comment-content"
    dangerouslySetInnerHTML={
      Object {
        "__html": undefined,
      }
    }
  />
</div>
`;
